service/application/src/main/java/com/solace/maas/ep/common/messages/CommandMessage.java

```diff
@@ -0,0 +1,43 @@
+package com.solace.maas.ep.common.messages;
+
+import com.solace.maas.ep.event.management.agent.plugin.command.model.CommandBundle;
+import com.solace.maas.ep.event.management.agent.plugin.mop.MOPMessage;
+import com.solace.maas.ep.event.management.agent.plugin.mop.MOPMessageType;
+import com.solace.maas.ep.event.management.agent.plugin.mop.MOPProtocol;
+import com.solace.maas.ep.event.management.agent.plugin.mop.MOPUHFlag;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class CommandMessage extends MOPMessage {
+
+    private String correlationId;
+    private String context;
+    private String messagingServiceId;
```

Comment:
Can we have this named as `serviceId` or some other generic name so it becomes more understandable when we use the command for schemaRegistries?

Reply:
Right, good point!